packages/base/rules-snapshot.json

```diff
@@ -83,6 +83,25 @@
   "getter-return": "error",
   "grouped-accessor-pairs": "error",
   "guard-for-in": "error",
+  "id-denylist": [
+    "error",
+    "err",
+    "cb",
+    "callback",
+    "req",
+    "res"
```

Comment:
The answer to the question “What do the people who use `req`/`res` instead of `request`/`response` do with all the time they save from typing the shorter names?” is “Occasionally wonder whether `res` means ‘result’ or ‘response’.”

Reply:
That said, the `(req, res)` pattern is so common, we should just keep it.